src/components/SiderMenu/BaseMenu.tsx

```diff
@@ -109,53 +123,42 @@ export default class BaseMenu extends PureComponent {
         <span>{name}</span>
       </Link>
     )
-  };
+  }
 
-  conversionPath = path => {
+  function conversionPath (path) {
     if (path && path.indexOf('http') === 0) {
       return path
     }
     return `/${path || ''}`.replace(/\/+/g, '/')
-  };
+  }
```

Comment:
this we can move to the module scope as well.

Reply:
done.